### PR TITLE
Use ninja for high level exawind packages since fortran was removed

### PR DESCRIPTION
**Pull request by jrood-nrel**
I haven't tested this yet.

### CODE REVIEW

**Comment by jrood-nrel** [comment]
This seems to work for me. Hoping for a better experience this time around without fortran.

File: repos/exawind/packages/amr-wind/package.py [hunk 1]
```diff
@@ -25,6 +25,9 @@ class AmrWind(bAmrWind):
     depends_on('hdf5+mpi', when='+hdf5+mpi')
     depends_on('h5z-zfp', when='+hdf5')
     depends_on('zfp', when='+hdf5')
+    depends_on("ninja", type="build")
+
+    generator = "Ninja"
```
Comment (psakievich) [inline repos/exawind/packages/amr-wind/package.py]:
```suggestion
    @property
    def generator(self):
          if '+ninja' in self.spec:
              return "Ninja"
          else:
              return "Unix Makefiles"
```
does this work to make ninja optional

File: repos/exawind/packages/amr-wind/package.py [hunk 2]
```diff
@@ -25,6 +25,9 @@ class AmrWind(bAmrWind):
     depends_on('hdf5+mpi', when='+hdf5+mpi')
     depends_on('h5z-zfp', when='+hdf5')
     depends_on('zfp', when='+hdf5')
+    depends_on("ninja", type="build")
```
Comment (psakievich) [inline repos/exawind/packages/amr-wind/package.py]:
```suggestion
    depends_on("ninja", type="build", when='+ninja')
    variant('ninja', default=False)
```

File: repos/exawind/packages/amr-wind/package.py (hunk 1 above)
Comment (jrood-nrel) [reply repos/exawind/packages/amr-wind/package.py]: Oh cool, yeah let me try that.

**Review by psakievich — APPROVED** [review]

**Pull request merged**